plv8.cc

```diff
@@ -134,6 +134,16 @@ static plv8_exec_env		   *exec_env_head = NULL;
 extern const unsigned char coffee_script_binary_data[];
 extern const unsigned char livescript_binary_data[];
 
+class Plv8ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
+ public:
+  virtual void* Allocate(size_t length) {
+    void* data = AllocateUninitialized(length);
+    return data == NULL ? data : memset(data, 0, length);
+  }
+  virtual void* AllocateUninitialized(size_t length) { return palloc(length); }
```

Comment:
No it's actually mandatory to prevent PG exception from leaking to C++ layer, I guess.  If you take a close look at other places including caller stack, we catch the PG exception and translate it to C++ pg_error exception before going back to C++, one way or another.  So I'd like you to do it.